server/middleware/populateReport.ts

```diff
@@ -0,0 +1,19 @@
+import type { NextFunction, Request, Response } from 'express'
+
+import logger from '../../logger'
+
+// eslint-disable-next-line import/prefer-default-export
+export function populateReport() {
+  return async (req: Request, res: Response, next: NextFunction): Promise<void> => {
+    try {
+      const { incidentReportingApi } = res.locals.apis
+      res.locals.incident = await incidentReportingApi.getReportById(req.params.id)
```

Comment:
i think we need to call reports “reports”, because incidents are actually _groups_ of reports